packages/frontend/src/components/refreshReminder.tsx

```diff
@@ -44,13 +44,17 @@ const RefreshReminder = () => {
                                         })
                                         const { transaction, error } =
                                             await userContext.getAirdrop()
+
+                                        if (error) throw new Error(error)
```

Comment:
in the future, can we set the error message in the frontend?
so that non-technical user can know the error message 😅

Reply:
Yes, logically I throw error here just wanna let the `queue` return a fail record, of course we can modify the function and return `error` as well to let the `queue` record it as a failed action. However to display to users, I think the current progress bar is too small to show error message, no idea where to put the error message now.